### PR TITLE
style: run prettier on entire code base

### PR DESCRIPTION
**Pull request by DSchau**
This PR--which does not necessarily need to be merged--runs prettier on the code base so we can unify code style.

This also fixes some issues where we were directly mutating state (i.e. `this.state =`) instead of `this.setState()` like [here](https://github.com/objectpartners/react-redux-timesheet/blob/master/final/src/components/employees/EmployeeForm.js#L42)

### CODE REVIEW

**Comment by meyerds** [comment]
Initial reaction: I'd want to increase the max line length prettier runs with. Seems like it's wrapping lines way earlier than it needs to.

**Comment by DSchau** [comment]
Yeah, I can bump it up to 100, I won't die on that hill. I do like using the defaults mostly--although I've already set single quote up, so it's not as if there's not a precedent for that already.

File: final/src/components/employees/EmployeeForm.js [hunk 1]
```diff
@@ -227,8 +207,10 @@ class EmployeeForm extends Component {
           </div>
           <FormControl.Feedback />
         </FormGroup>
-
-        <Button bsStyle="success" onClick={this.handleSave} disabled={!this.validateAll()}> Save </Button>&nbsp;
+        <Button bsStyle="success" onClick={this.handleSave} disabled={!this.validateAll()}>
+          {' '}
```
Comment (meyerds) [inline final/src/components/employees/EmployeeForm.js]: this is interesting. shouldn't whitespace collapse to a single space anyway, without the explicit `{' '}`? Or does JSX work differently than browsers/HTML?

File: final/src/components/timesheets/TimesheetRow.js [hunk 2]
```diff
@@ -1,47 +1,56 @@
...
   showDetail(timesheet) {
-    if(timesheet.deleted) {
+    if (timesheet.deleted) {
       console.log('You cannot edit a deleted timesheet.');
       return;
     }
 
-    this.props.history.push('/employees/' + timesheet.user_id + '/timesheets/detail/' + timesheet._id);
+    this.props.history.push(
+      '/employees/' + timesheet.user_id + '/timesheets/detail/' + timesheet._id
```
Comment (meyerds) [inline final/src/components/timesheets/TimesheetRow.js]: I guess prettier doesn't automatically convert to template strings eh? ;-)

Comment (DSchau) [reply final/src/components/timesheets/TimesheetRow.js]: ugh I wish. That'd probably be a fairly brittle thing to update I'd guess tho?

File: lab-03-routing/src/components/timesheets/Timesheets.js [hunk 3]
```diff
@@ -1,155 +1,374 @@
...
     this.state = {
       pageConfig: {
         data: [
           {
-            "name": "UserOne",
-            "beginDate": "2013-11-18T00:00:00.000Z",
-            "endDate": "2013-11-24T00:00:00.000Z",
-            "description": "Timesheet one for user",
-            "timeunits": [
-              {"dateWorked": "2013-11-18T00:00:00.000Z", "hoursWorked": 8, "project": "Project1"},
```
Comment (meyerds) [inline lab-03-routing/src/components/timesheets/Timesheets.js]: I wish there were a way to tell Prettier to leave these alone. Breaking it up over multiple lines just makes this chunk of test data harder to manage IMO.

Comment (DSchau) [reply lab-03-routing/src/components/timesheets/Timesheets.js]: There is!

```
// prettier-ignore
```

I believe.

**Comment by tmartensen** [comment]
Too many semicolons in here. Otherwise, this LGTM. 😉

**Pull request merged**